libhttpsimple-bindings/src/main/scala/libhttpsimple/LibHttpSimple.scala

```diff
@@ -158,12 +158,11 @@ object LibHttpSimple {
       result
     }
 
-  private def httpHeadersToDelimitedString(headers: Map[String, String]): String =
+  private def httpHeadersToDelimitedString(headers: Map[String, String]): Seq[String] =
```

Comment:
A bit of a Scala wart here - could you add `import scala.collection.immutable.Seq` at the top of the file? The default `Seq` isn't immutable (to support varg syntax with arrays), but will be in a future release AFAIK.